app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {}
+
```

Comment:
you don't need this line here, because you are creating this variable later. This line has no impact.